config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to nothing/blank (i.e. elasticsearch.requestHeadersWhitelist: )
+# or the empty list (i.e. elasticsearch.requestHeadersWhitelist: [])
```

Comment:
If nothing else, for consistency sake I think we should stick to `[]`.  I'm sure there are other configurations in kibana.yml that technically would allow an empty value, but we don't advertise it as such.  Your call though.